src/Collapse.js

```diff
@@ -81,14 +78,15 @@ const Collapse = React.createClass({
 
 
   getMotionHeight(height) {
-    const {isOpened, springConfig} = this.props;
+    const {isOpened, springConfig, fixedHeight} = this.props;
```

Comment:
I am wrong here, sorry. `getMotionHeight` is used for both fixed and dynamic height.